spark-template-velocity/src/main/java/spark/template/velocity/VelocityTemplateEngine.java

```diff
@@ -63,7 +105,13 @@ public VelocityTemplateEngine(VelocityEngine velocityEngine) {
      */
     @Override
     public String render(ModelAndView modelAndView) {
-        Template template = velocityEngine.getTemplate(modelAndView.getViewName());
+    	Template template;
+    	if(encoding != null || !encoding.isEmpty()) {
+    		template = velocityEngine.getTemplate(modelAndView.getViewName(), encoding);
```

Comment:
Please use the Spark code style: https://github.com/perwendel/spark/tree/master/config